File: easy/q9/q9.rs
impl Solution {
    pub fn is_palindrome(x: i32) -> bool {
        let s = x.to_string();
        let array: Vec<char> = s.chars().collect();

        for i in 0..array.len() / 2 {
            if array[i] != array[array.len() - 1 - i] {
                return false;
            }
        }

        true
    }
}

File: easy/q9/q9.cpp
#include <string>
using namespace std;
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x);
        const char* array = s.c_str();

        for (int i=0; i< s.length(); i++){
           if (array[i] != array[s.length() - 1 - i]) return false;
     
        }
        return true;
    }
};